resp/resp3/resp.go

```diff
@@ -1609,6 +1609,8 @@ func saneDefault(prefix Prefix) (interface{}, error) {
 	// we don't handle ErrorPrefix because that always returns an error and
 	// doesn't touch I
```

Comment:
This comment might need some addition as errors will be handled here in case of `TreatErrorsAsValues`.

Reply:
The comment doesn't make much sense with the new code so I removed it.